pipenv/cli.py

```diff
@@ -1251,7 +1251,7 @@ def update(dev=False, three=None, python=None, dry_run=False, bare=False, dont_u
     ensure_project(three=three, python=python, validate=False)
     # --dry-run
     if dry_run:
-        dont_upgrade = True
+        # dont_upgrade = True
```

Comment:
@kennethreitz this variable is assigned but never used.  I commented it out for now, but do you intend to use it going forward or can it be removed?